src/algebra/floor.lean

```diff
@@ -115,6 +116,9 @@ begin
   exact ⟨floor_le v, lt_floor_add_one v⟩
 end
 
+lemma floor_eq_on_Ico (n : ℤ) : ∀ x ∈ (set.Ico n (n+1) : set α), (n : α) = floor x :=
```

Comment:
Is there a reason to put the complicated version on the right-hand side? Did you need to rewrite in this direction?

Reply:
It's a bit easier to use with e.g `tendsto_nhds_within_congr`, which assumes you now the limit of the left side. But that's not that important, reversing this seems like the better option.